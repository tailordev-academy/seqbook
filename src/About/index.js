import React from 'react';

const About = () => (
  <div className="About">
    <div className="container-fluid">
      <p>
        This project has been made during a TailorDev Academy session in order
        to learn React and its ecosystem.
      </p>
    </div>
  </div>
);

export default About;
