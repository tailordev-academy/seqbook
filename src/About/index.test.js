import React from 'react';
import renderer from 'react-test-renderer';

import About from './index';

it('renders correctly', () => {
  const tree = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
