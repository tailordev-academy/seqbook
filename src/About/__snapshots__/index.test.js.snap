// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="About"
>
  <div
    className="container-fluid"
  >
    <p>
      This project has been made during a TailorDev Academy session in order to learn React and its ecosystem.
    </p>
  </div>
</div>
`;
